roles/foreman_installer/tasks/installer_pr.yml

```diff
@@ -0,0 +1,17 @@
+---
+- name: "install git"
+  package:
+    name: 'git'
+    state: 'present'
+  tags:
+    - packages
+
+- name: "fetch git PR"
+  shell: >
+      git init &&
+      git remote add {{ installer_fork }}  https://github.com/{{ installer_fork }}/foreman-installer.git &&
```

Comment:
Should this be simplified to `git clone --branch {{ installer_branch }} https://github.com/{{ installer_fork }}/foreman-installer.git /usr/share/foreman-installer`

Reply:
Given the destination folder already exists, this will fail.

Reply:
Right. I think it may make sense to replace the package altogether and do a pure source installation. Right now it doesn't pull in the modules either. While that's slower, it is the correct thing to do. In a PR you can modify the Puppetfile to point to a branch for complete testing.